### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by santtosr** [tiberiuzuld/angular-gridster2#825]
How can I hide the component's vertical scroll and display only the one on the main screen

**Comment by jasonparallel** [comment]
I don't think this  is a gridster question. You control the content in the item. Use css to make it behave how you want. overflow:hidden? https://www.w3schools.com/cssref/pr_pos_overflow.asp
